### PR TITLE
Show Practice create errors on practice view

### PR DESCRIPTION
**Pull request by 7imbrook**
Changes:
- adds pinning to error reducer

### CODE REVIEW

File: app/models/wordinfo.rb [hunk 1]
```diff
@@ -15,21 +15,19 @@ class Wordinfo < ApplicationRecord
...
+        { forms: { only: [:word, :part_of_speech] } },
+        { synonyms: { only: [:word] } },
+        { antonyms: { only: [:word] } },
+        { sentences: { only: [:context_sentence] } }
+      ],
+      except: [:id, :user_id, :lesson_id, :created_at, :updated_at]
+    ))
+    wordinfos&['synonyms']&.map! { |synonym| synonym['word'] }
+    wordinfos&['antonyms']&.map! { |antonym| antonym['word'] }
+    wordinfos&['context_sentence']&.map! { |ct| ct['context_sentence'] }
```
Comment (mwitte98) [inline app/models/wordinfo.rb]: Please change syntax to `wordinfos&.[]('')` and undo changes on lines 18-27

File: client/src/actions/practice.ts [hunk 2]
```diff
@@ -26,8 +31,10 @@ export function generatePractice(id: number, type: SectionType) {
             dispatch({ type: 'practice_save_local', practice, id });
         })
         .catch((err: Error) => {
+            console.log(err);
```
Comment (mwitte98) [inline client/src/actions/practice.ts]: console.log

**Review by mwitte98 — CHANGES_REQUESTED** [review]

File: app/models/wordinfo.rb (hunk 1 above)
Comment (7imbrook) [reply app/models/wordinfo.rb]: I'm really not sure what's suppose to be here, can you just push it up to my branch. It was a merge conflict

Comment (mwitte98) [reply app/models/wordinfo.rb]: Done

**Comment by mwitte98** [comment]
@7imbrook I just remembered that the reason why I kept wordinfo sentences formatted as `[{context_sentence: ''}]` and not an array of strings is to support primary and secondary sentences. I know it's not on the short term roadmap, but we should leave it as it is now so we don't have to change it back to an array of objects later.

**Comment by djrenren** [comment]
Couldn't we just use a different context sentence?

**Comment by mwitte98** [comment]
We could. But we wanted all students to get the same sentence/questions when quizzed for fairness

**Comment by djrenren** [comment]
Yeah alternates need to be the same on the question model but we don't need
them to be explicitly called out in the lesson

**Comment by mwitte98** [comment]
Okay, I can see how that would work. I'll need to complete #205 first before merging this PR since those are the tests that are failing.

**Comment by mwitte98** [comment]
Removed the changes to the 2 ruby files in this PR since I changed sentences to an array of strings throughout in #206

**Review by mwitte98 — APPROVED** [review]

**Pull request merged**